protobuf/daemon/settings.proto

```diff
@@ -27,7 +27,6 @@ message Settings {
   bool firewall = 2;
   bool kill_switch = 3;
   AutoconnectData auto_connect_data = 4;
-  bool ipv6 = 5;
```

Comment:
This requires bump of the version here https://github.com/NordSecurity/nordvpn-linux/blob/main/protobuf/daemon/common.proto#L9 and also updates on GUI side

Reply:
Nope, bump not needed. My bad. But we have ipv6 settings reference in daemon mock